test/jquery.js

```diff
@@ -21,12 +22,13 @@
 	// Define configuration parameters controlling how jQuery is loaded
 	if ( QUnit ) {
 
-		// AMD loading is asynchronous and incompatible with synchronous test loading in Karma
+		// ES modules loading is asynchronous and incompatible with synchronous
+		// test loading in Karma.
 		if ( !window.__karma__ ) {
 			QUnit.config.urlConfig.push( {
-				id: "amd",
-				label: "Load with AMD",
-				tooltip: "Load the AMD jQuery file (and its dependencies)"
+				id: "jsmodules",
```

Comment:
What do you think about this parameter name?

Reply:
Perhaps "esmodules" to make it more clear that the modules are standard es format.

Reply:
Renamed.